packages/kbn-unified-data-table/src/utils/get_render_cell_value.tsx

```diff
@@ -20,38 +20,61 @@ import {
   EuiFlexGroup,
   EuiFlexItem,
 } from '@elastic/eui';
-import { FieldFormatsStart } from '@kbn/field-formats-plugin/public';
+import type { FieldFormatsStart } from '@kbn/field-formats-plugin/public';
 import type {
   DataTableRecord,
   EsHitRecord,
   ShouldShowFieldInTableHandler,
 } from '@kbn/discover-utils/types';
-import { MAX_DOC_FIELDS_DISPLAYED, formatFieldValue, formatHit } from '@kbn/discover-utils';
-import { DiscoverGridContext } from './discover_grid_context';
-import { JsonCodeEditor } from '../json_code_editor/json_code_editor';
-import { defaultMonacoEditorWidth } from './constants';
-import { useDiscoverServices } from '../../hooks/use_discover_services';
+import { formatFieldValue, formatHit } from '@kbn/discover-utils';
+import { UnifiedDataTableContext } from '../table_context';
+import { JsonCodeEditor } from '../components/json_code_editor/json_code_editor';
```

Comment:
Related to the comment about the height of the JSON tab in the DocViewer, there's seems to be an height issue when expanding the Document column:

<img width="1817" alt="Discover_-_Elastic_und_Discover_-_Elastic" src="https://github.com/elastic/kibana/assets/463851/d610c518-876c-4dcf-a3b5-e50865e74df7">

Reply:
Great catch! Fixed in the commit https://github.com/elastic/kibana/pull/163211/commits/6a0abe07e648f7f3112e8747c69a09eb31579952. Looks like code editor was exposed the prop, but never use it :sweat_smile:

Reply:
Confirmed it was fixed, thx a lot!